### PR TITLE
Add Search Feature

### PR DESCRIPTION
**Pull request by dianewh**
Added Search feature to filter reviews shown in ReviewList to only display reviews containing searched term.  Feature also works when no reviews are found and includes a function to return to displaying all reviews. Added some basic styling to Search using CSS Modules. 

@dwgl80 @ArthurHovanesian @edwardbaeg Please review, thanks!

### CODE REVIEW

File: client/src/components/ReviewList.jsx [hunk 1]
```diff
@@ -10,20 +10,63 @@ class ReviewList extends React.Component {
 		};
 	}
 	render() {
```
Comment (ArthurHovanesian) [inline client/src/components/ReviewList.jsx]: For readability, could you leave an empty space in-between the constructor and render functions?

File: client/src/index.jsx [hunk 2]
```diff
@@ -33,10 +33,50 @@ class App extends React.Component {
...
+	}
+
+	showAllReviews () {
+		this.setState({
+			showSearch: false,
+			searchResults: [],
+		})
+	}
 	
   calculateAvgRating (reviews) {
```
Comment (ArthurHovanesian) [inline client/src/index.jsx]: Is the *100/100 necessary here?

**Comment by ArthurHovanesian** [comment]
Oh one more thing! Is your search bar case-sensitive and if so, could yo refactor so it's not?

**Comment by dianewh** [comment]
@ArthurHovanesian Thanks for the comments! Added the suggested changes.

**Pull request merged**